portfolio/src/main/webapp/script.js

```diff
@@ -52,4 +52,25 @@ function addName(name) {
     console.log('Adding name to DOM...');
     const nameButton = document.getElementById('name-container');
     nameButton.innerText = name;
+}
+
+function fetchComments() {
+    fetch('/data').then(response => response.json()).then((comments) => {
+        console.log('Fetching comments...');
+        console.log(comments);
+        const commentListElement = document.getElementById('comment-container');
+        commentListElement.innerHTML = '';
+        commentListElement.appendChild(
+            createListElement('Comment 1: ' + comments[0]));
+        commentListElement.appendChild(
+            createListElement('Comment 2: ' + comments[1]));
+        commentListElement.appendChild(
+            createListElement('Comment 3: ' + comments[2]));
```

Comment:
What happens if the server changes to return more comments? Or fewer comments?

Might be good to use a loop here instead of assuming a hard coded number of comments and that way the javascript can be ready for a variable number of comments.